test/test_quantization.py

```diff
@@ -261,6 +261,166 @@ def checkQuantized(model):
         model = quantize(ManualQuantModel().eval(), test_only_eval_fn, self.calib_data)
         checkQuantized(model)
 
+class PostTrainingDynamicQuantTest(QuantizationTestCase):
+    def test_single_layer_dynamic(self):
+        r"""Dynamic Quantize SingleLayerLinearModel which has one Linear module,
+        make sure it is swapped to nnqd.Linear which is the quantized version of
+        the module
+        """
+        model = SingleLayerLinearModel().eval()
+        qconfig_dict = {
+            '': default_qconfig
```

Comment:
Please add a default_qconfig for dynamic quant that does not have activation observers

Reply:
This doesn't work: the activation observers are the required positional argument for QConfig in https://github.com/pytorch/pytorch/blob/master/torch/quantization/QConfig.py#L6-L7.